rules-java-diva/addon/src/main/java/org/jboss/windup/rules/apps/diva/analysis/JanusGraphReport.java

```diff
@@ -0,0 +1,193 @@
+package org.jboss.windup.rules.apps.diva.analysis;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.WindupVertexFrame;
+import org.jboss.windup.graph.service.GraphService;
+import org.jboss.windup.rules.apps.diva.model.DivaConstraintModel;
+import org.jboss.windup.rules.apps.diva.model.DivaContextModel;
+import org.jboss.windup.rules.apps.diva.model.DivaOpModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRequestParamModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRestCallOpModel;
+import org.jboss.windup.rules.apps.diva.model.DivaSqlOpModel;
+import org.jboss.windup.rules.apps.diva.model.DivaStackTraceModel;
+import org.jboss.windup.rules.apps.diva.model.DivaTxModel;
+import org.jboss.windup.rules.apps.diva.service.DivaStackTraceService;
+import org.jboss.windup.rules.apps.java.model.JavaClassModel;
+import org.jboss.windup.rules.apps.java.model.JavaMethodModel;
+import org.jboss.windup.rules.apps.java.service.JavaClassService;
+import org.jboss.windup.rules.apps.java.service.JavaMethodService;
+
+import com.ibm.wala.classLoader.IMethod;
+import com.ibm.wala.classLoader.IMethod.SourcePosition;
+import com.ibm.wala.shrikeCT.InvalidClassFileException;
+import com.ibm.wala.types.MethodReference;
+import com.ibm.wala.util.strings.StringStuff;
+
+import io.tackle.diva.Report;
+import io.tackle.diva.Trace;
+
+public class JanusGraphReport<T extends WindupVertexFrame> implements Report {
```

Comment:
`JanusGraphReport` is too generic name and better change it to reflect its usage within the Diva context, so something like `DivaReport` might be better.
At the same time the suffix `Report` confuses me: isn't the Diva report supposed to be in https://github.com/windup/windup/pull/1434?